### ISSUE TITLE
please add button style

### ISSUE THREAD

**Issue by IrfanH495** [hrydgard/ppsspp#14515]
### What should happen?
can add button style like this
![pp](https://user-images.githubusercontent.com/78854859/120984773-26d5a900-c7a5-11eb-961c-e7a24a9a47a3.png)

maybe it would be better if there was a custom skin pad option like epsxe [ but it will take a long time for developer, to add options Costume skin Pad ]
![Screenshot_20210607-153919](https://user-images.githubusercontent.com/78854859/120986412-c3e51180-c7a6-11eb-8972-502afb6e84b5.png)

#### Checklist

 - [ ] Check the [latest git build](https://buildbot.orphis.net/ppsspp/) in case it's already implemented.
 - [ ] [Search for other requests](https://github.com/hrydgard/ppsspp/search?q=my+issue&type=issues) of the same feature.

**Comment by Gamemulatorer** [comment]
I also mention this in https://github.com/hrydgard/ppsspp/issues/5624#issuecomment-434106868 but it's not devs priority as of now.

**Comment by IrfanH495** [comment]
> I also mention this in [#5624 (comment)](https://github.com/hrydgard/ppsspp/issues/5624#issuecomment-434106868) but it's not devs priority as of now.

So will this issue be closed?

**Comment by hrydgard** [comment]
It's a valid feature request, unless it's a duplicate I'll leave it open. Might take a long time to get to it though given other priorities...

**Comment by iota97** [comment]
For this use case #13869 can somewhat work around that by adding the new shape in the atlas and then adding it to GamepadEmu.h, still probably going to take quite a while and not a definitive solution at all for button style.

**Comment by hrydgard** [comment]
Long term I plan to move away from pre-baked image atlases except for the basic font rendering. Will make it simpler to implement stuff like this.

**Comment by IrfanH495** [comment]
> For this use case #13869 can somewhat work around that by adding the new shape in the atlas and then adding it to GamepadEmu.h, still probably going to take quite a while and not a definitive solution at all for button style.

for the ppsspp you share, it's very good.
but it's better if you can add 2 buttons in one circle for tekken game
Example
![1623062848-picsay](https://user-images.githubusercontent.com/78854859/121004586-116a7a00-c7b9-11eb-9c8f-7fdf95fa5f89.png)

**Comment by Gamemulatorer** [comment]
> > For this use case #13869 can somewhat work around that by adding the new shape in the atlas and then adding it to GamepadEmu.h, still probably going to take quite a while and not a definitive solution at all for button style.
> 
> for the ppsspp you share, it's very good.
> but it's better if you can add 2 buttons in one circle for tekken game
> Example
> ![1623062848-picsay](https://user-images.githubusercontent.com/78854859/121004586-116a7a00-c7b9-11eb-9c8f-7fdf95fa5f89.png)

No need for that PPSSPP support combo key

**Comment by IrfanH495** [comment]
@Gamemulatorer 
yes I know, but if it only uses the number 1/2 I like to forget what key combination it is..

**Comment by Leopard20** [comment]
A square shape (instead of just rectangles) will be neat too.

**Comment by ResurrectionRedmi9** [comment]
@iota97 can you do this?
Make the button glow and at the same time enlarge?
Hope it will implemented in v1.12 😁

https://user-images.githubusercontent.com/90336524/134271304-209f8cc2-4305-4a2a-a21a-e37083613147.mp4

**Comment by IrfanH495** [comment]
Thanks this is what i mean
#21535
